spec/services/attribute_asserter_spec.rb

```diff
@@ -703,6 +719,44 @@
       end
     end
 
+    context 'when biometric IAL preferred is requested' do
+      let(:subject) do
+        described_class.new(
+          user: user_under_test,
+          name_id_format: name_id_format,
+          service_provider: service_provider,
+          authn_request: biometric_preferred_ial_authnrequest,
+          decrypted_pii: decrypted_pii,
+          user_session: user_session,
+        )
+      end
+      context 'when the user has been proofed with biometric comparison' do
+        let(:user_under_test) { biometric_verified_user }
```

Comment:
what if we just called it `user`?
```suggestion
        let(:user) { biometric_verified_user }
```

Reply:
I was unhappy with inconsistencies in the naming of some of the original fixture objects, e.g. `user` vs `ial1_user`. I may end up renaming the original `user` to `proofed_but_not_biometric_verified_user` or something more concise, in which case `user` becomes available.

Reply:
rspec uses the most specific context/describe block to define a fixture. so when you define `user` here, it will be used automatically without having to change anything about more general fixtures. 

if you'd prefer to use a different name, that's fine but `user_under_test` is confusing, so please don't use that.

Reply:
Reverted to `user`